sdk/communication/AzureCommunication/Source/CommunicationIdentifierSerializer.swift

```diff
@@ -99,6 +72,55 @@ public class CommunicationIdentifierSerializer {
 
         return CommunicationCloudEnvironment(environmentValue: model.requestString);
     }
+    
+    static func assertOneNestedModel(_ identifier: CommunicationIdentifierModel) throws {
+        var presentProperties = [String]()
```

Comment:
I think presentProperties is not being returned.  Can we just use an integer to count if identifier has more than 1 communicationUser , phoneNumber and MicrosoftTeamsUser

Reply:
Yes that would make sense and simplify it a bit